server/container/guice/cassandra-rabbitmq-guice/src/test/java/org/apache/james/WithEncryptedSwiftExtension.java

```diff
@@ -1,72 +0,0 @@
-/*
```

Comment:
Do we have some tests about encrypted S3 ?

Reply:
No

Reply:
(I read the description later, sorry for useless comment)